### ISSUE TITLE
sccache 0.2.7 no longer installs from crates.io

### ISSUE THREAD

**Issue by neoeinstein** [mozilla/sccache#355]
sccache 0.2.7 depends on ring ^0.12.1, but all compatible versions of the ring crate have been yanked. Because of this, it is now impossible to install sccache 0.2.7 using `cargo install` from crates.io.

**Comment by luser** [comment]
Ah, this is unfortunate! I've been putting off shipping a new release because we had a bunch of in-progress work but I guess I should just do it to fix this issue.

**Comment by thomaseizinger** [comment]
Hi @luser, is there an ETA on when this is going to be fixed? We use/rely on `sccache` in our CI builds to (greatly) speed up the build.

**Comment by petemill** [comment]
Since we're not getting a timely response on fixing cargo install and I'm having issues with current master, I've created a branch of 0.2.7 which points at the compatible version of ring - https://github.com/mozilla/sccache/compare/0.2.7...petemill:0.2.7-fix-ring-dep

You can install from cli:
```
cargo install --git https://github.com/petemill/sccache.git --rev 5ce26c30c16ecd14ff16ec53bb242340ca402423
```

@luser can we get a small 0.2.8 publish using the above?

**Comment by luser** [comment]
Apologies, there was terrible timing on all of this as I was out of the country for several days. I don't see any showstopper issues that have been filed recently, or any open PRs that seem like they absolutely need to be landed before a 0.2.8 so I'll just tag and release what's currently on master today.

**Comment by luser** [comment]
We still have a patch for `tiny_http` in Cargo.toml which blocks releasing *directly* from master:
https://github.com/tiny-http/tiny-http/pull/151#issuecomment-456850875

I've [created a branch](https://github.com/mozilla/sccache/tree/0.2.8-workaround) and [added a commit that removes that patch](https://github.com/mozilla/sccache/commit/978813cef19161ced6be33da517f8c4feac060dc) on the branch and will release 0.2.8 from that branch.

**Comment by luser** [comment]
Released 0.2.8:
* https://crates.io/crates/sccache/0.2.8
* https://github.com/mozilla/sccache/releases/tag/0.2.8

Sorry for all the inconvenience, but thanks to all of you for using sccache and letting us know about your problems!
